esphome/components/pid/climate.py

```diff
@@ -27,6 +28,17 @@
 CONF_NEGATIVE_OUTPUT = "negative_output"
 CONF_MIN_INTEGRAL = "min_integral"
 CONF_MAX_INTEGRAL = "max_integral"
+CONF_OUTPUT_SAMPLES = "output_samples"
```

Comment:
```suggestion
CONF_OUTPUT_AVERAGING_SAMPLES = "output_averaging_samples"
```

I don't really like the name, as it doesn't tell you what it actually does. Perhaps adding `averaging` is better?